azure-pipelines.yml

```
@@ -97,7 +98,10 @@ jobs:
 
   - script: |
       eval $(opam env)
+      export OCAMLPATH=$(AZURE_BUILDIR):"$OCAMLPATH"
```

```suggestion
      export OCAMLPATH=$AZURE_BUILDIR:"$OCAMLPATH"
```
or
```suggestion
      export OCAMLPATH=$(Agent.BuildDirectory):"$OCAMLPATH"
```
(without an `env` declaration, same syntax as the -prefix above)